dev_scripts_helpers/documentation/latex_abbrevs.sty

```diff
@@ -1,5 +1,14 @@
 % From https://tex.stackexchange.com/questions/75667/change-colour-on-chapter-section-headings-lyx
 \usepackage{xcolor}
+
```

Comment:
There are multiple latex_abbrevs in //tutorials and //umd_classes (we should remove these multiple copies, but Latex is horrible at handling dirs). Can you copy / paste the same changes around?